### ISSUE TITLE
Comparison table

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/ky#19]
In the future, when Ky is more mature, we should add a comparison table to the readme. Like https://github.com/sindresorhus/got#comparison

**Comment by Airkro** [comment]
https://github.com/sindresorhus/got/pull/895

**Comment by sholladay** [comment]
To avoid duplicating information, would it make sense to simply link to `got`'s comparison table, (which includes Ky)?

**Comment by szmarczak** [comment]
I don't think so. Ky should be comparable to other fetch-based clients, not to Node ones.

**Comment by sindresorhus** [comment]
Closing as I don't really want to maintain this. We can rather be good at selling Ky in the "benefits" section.
